src/chat.cpp

```diff
@@ -263,15 +276,22 @@ u32 ChatBuffer::formatChatLine(const ChatLine& line, u32 cols,
 	//EnrichedString line_text(line.text);
 
 	next_line.first = true;
-	bool text_processing = false;
+	bool mark_newline = false;
```

Comment:
Would you mind adding a short comment what this boolean does? Does it force a newline on the next fragment? Does it split (wrap) fragments?